docs/operations/shipperctl.rst

```diff
@@ -78,6 +84,9 @@ Here is something more interesting: having 2 application clusters, and marking o
     scheduler:
       unschedulable: true
 
+Using Contexts That Are Not Valid As a Kubernetes Name
```

Comment:
Consider renaming this title: personally, I found it quite confusing. Example: Workaround for GKE cluster naming convention conflict.